### PR TITLE
Time based automation for screens

### PR DESCRIPTION
**Pull request by nerrehmit**
changed a screen resource to lowercase and added a new time based automation for the switches category to manage the office screens

### CODE REVIEW

File: automations.yaml [hunk 1]
```diff
@@ -71,7 +69,35 @@
...
+      entity_id:
+      - switch.systvscreen
+      - switch.kafiscreen
+
+- alias: Office Screens Off
+  trigger:
+  - platform: time
+    at: '18:00:00'
+  condition:
+    condition: state
```
Comment (bliemli) [inline automations.yaml:93]:
```suggestion
  - condition: state
```

File: automations.yaml [hunk 2]
```diff
@@ -71,7 +69,35 @@
...
+    entity_id: 'binary_sensor.workday_sensor_be'
+    state: 'on'
+  action:
+    service: switch.turn_off
```
Comment (bliemli) [inline automations.yaml:97]:
```suggestion
  - service: switch.turn_off
```

File: automations.yaml [hunk 3]
```diff
@@ -71,7 +69,35 @@
...
+- alias: Office Screens On
+  trigger:
+  - platform: time
+    at: '7:45:00'
+  condition:
+  - condition: state
+    entity_id: 'binary_sensor.workday_sensor_be'
+    state: 'on'
+  action:
+    service: switch.turn_on
```
Comment (bliemli) [inline automations.yaml:82]:
```suggestion
  - service: switch.turn_on
```

**Review by bliemli — CHANGES_REQUESTED** [review]

**Comment by nerrehmit** [comment]
Thanks for reviewing, we can add the dashes in the proposed locations. However I just copied the existing structure from the "Office Lights on/off" so I would suggest to either leave it like it is or add them everywhere. What do you think?

**Comment by bliemli** [comment]
I'll add them everywhere. Thanks for your PR!

**Pull request merged**